src/ai/backend/test/cli_integration/conftest.py

```diff
@@ -140,6 +141,16 @@ def keypair_resource_policy() -> str:
     return fake.unique.word()
 
 
+@pytest.fixture
+def make_txt_file() -> Generator[TextIOWrapper, None, None]:
+    FILE_NAME = "test.txt"
+    with open(FILE_NAME, "w") as f:
+        f.write("This file is for testing.")
+    yield f
+
+    os.remove(FILE_NAME)
```

Comment:
Please use lowercases for local variables.
```suggestion
    filepath = "test.txt"
    with open(filepath, "w") as f:
        f.write("This file is for testing.")
    yield f

    os.remove(filepath)
```